### DISCUSSION TITLE
Was working on my 38" 21:9 LG monitor till I changed my MacBookPro with the new M2

### DISCUSSION THREAD

**Discussion by acomper** [MonitorControl/MonitorControl#1339] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

The app was working fine using a LG Monito 38" 21:9 with my MacBookPro Intel 2019, but not anymore with my new MacBookPro M2 Max: only the brightness control works, not the volume one.

### Expected behavior

Not able
![Screenshot 2023-03-17 alle 10 12 19](https://user-images.githubusercontent.com/128135592/225862445-98259259-9c3f-4cfa-befc-24dc0f95bcd3.png)
 to flag the Volume control

### Environment Information (please complete the following information)

```markdown
- macOS version:
- Mac model:
- MonitorControl version:
- Monitor(s):
- Monitor Cable(s)/Connection(s):
- Apple Silicon/M1 (yes or no):
```

**Comment by digicide** [comment]
Same for me, upgraded a 2019 Intel to an M2, now my Gigabyte M32U has only software dimming available and no volume.

**Comment by waydabber** [comment]
Take a look at this:

https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting#known-incompatibilities
